Mar 7, 2023 CSE Release Notes (#1811)
SumoLogic/sumologic-documentation: blog-cse/2023-03-07-application-update.md

---
title: March 7, 2023 Application Update
hide_table_of_contents: false
keywords:
  - sumo logic
  - service release notes
  - cloud siem
  - cse
tags: [cloud siem release notes, application update]
authors:
  - url: https://help.sumologic.com/release-notes-cse/rss.xml
    image_url: /img/release-notes/rss-orange.png
---

import useBaseUrl from '@docusaurus/useBaseUrl';

### Entity Relationship Graph

We are excited to announce the new Entity Relationship Graph. With this feature, you can now see a graphical visualization of all related Entities in an Insight, as well as additional relationships beyond the Insight. This enables you to more quickly understand relationships among Entities and the larger context behind a potential security threat.

:::note

> This feature is available to all customers but is currently in **Beta**. If you encounter any issues with this feature, please report them to Sumo Logic Support. Thank you!

Reviewer: ```suggestion
This feature is available to all customers but is currently in **Beta**. If you encounter any issues with this feature,  report them to Sumo Logic Support. We appreciate your feedback.
```
